apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableLikeToContainCheckerSamples.kt

```diff
@@ -27,4 +27,27 @@ class IterableLikeToContainCheckerSamples {
             }
         }
     }
+
+    @Test
+    fun atMost() {
+        expect(listOf("A,A,B,C,A,B,B")).toContain.inAnyOrder.atMost(2).entry{
+            toEqual("C")
+        }
+
+        expect(listOf(1,2,3,2,3,3)).toContain.inAnyOrder.atMost(2).entry{
+            toBeLessThanOrEqualTo(2)
+        }
+
+        fails {
```

Comment:
same same would be nice if you could explain why it fails.

*Bonus* add also an explanation why it fails in the method `exactly`